Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: all gen build edit open
+
+all: gen build
+
+gen:
+	@echo "Generating"
+	go run ./temporalcli/internal/cmd/gen-commands
```

Comment:
Cool. Updating. 

The Makefile really is only because I can't remember things. I can leave it as part of the patch but was intending to remove it.

The reason there are two targets for open and edit is that I couldn't remember which one I'd defined so I added both.